### ISSUE TITLE
TK Widths- should not be flush on the outside of partitions, should be in the middle of the partition

### ISSUE THREAD

**Issue by TEDSTANDAGE** [CCDevNelson/Snap-Updates#20]

**Comment by ClassyTeddy** [comment]
I believe I fixed the issue so now toe kicks are in the middle of the partition. Again, please use the TeddyChanges Branch to test it. I did have a few questions about toe kicks. Are they always 2.5 inches tall? Or should I make them match the height of the partitions? I just noticed that when I placed a toe kick on a hanging partition, it would place the toe kick on the floor and I would have to adjust the hanging height of the hanging partition to get it to line up.

**Comment by TEDSTANDAGE** [comment]
Toe kicks are more complex and come in various heights--we should have an in depth convo about them that can't happen over messages.

**Comment by TEDSTANDAGE** [comment]
Still flush:

![image](https://user-images.githubusercontent.com/61025894/77955322-d4395c80-7284-11ea-8a76-0946b269d7f1.png)

Teddy, give me a call when you get a chance

**Comment by ClassyTeddy** [comment]
I fixed them to be the proper 3/8 of an inch inset on both sides. Please test on your end.

**Comment by TEDSTANDAGE** [comment]
Still not able to get it to show up on my end

![image](https://user-images.githubusercontent.com/61025894/77960095-8cb6ce80-728c-11ea-93ca-9637487f092e.png)

**Comment by ClassyTeddy** [comment]
Can you post a screenshot of the history tab for the branch? I'm just wondering if the changes got sent through. I can see them on my end. Could you also try resetting your SNaP defaults?

**Comment by TEDSTANDAGE** [comment]
I looked at the History and didn't see any changes to the TKs so I fetched another version of your branch while that tab was opened and the changes came through (not sure why they didn't before--on either my laptop or office computer). It is now working though and I am getting the 3/8" inset on each side, at least for simple TKs. We will have a lot more work to do with TKs but this is a good initial change.

![image](https://user-images.githubusercontent.com/61025894/77962516-97736280-7290-11ea-9a9c-f19f6fd14cac.png)
